ci/prow/boskos/deployments/metrics.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: v1
+kind: Service
```

Comment:
this will be only exposed cluster clocally. Is that intended? If we want to read the values here from externally, we need to enable the loadbalancer type and with probably a static IP

Reply:
Oh, that's what that was supposed to do. I love the naming.

Reply:
No, I don't think it's necessary, but could be added later if desired.